include/spirv-tools/libspirv.h

```diff
@@ -271,6 +271,12 @@ typedef enum spv_binary_to_text_options_t {
   SPV_FORCE_32_BIT_ENUM(spv_binary_to_text_options_t)
 } spv_binary_to_text_options_t;
 
+typedef enum spv_linker_options_t {
+  SPV_LINKER_OPTION_NONE = SPV_BIT(0),
+  SPV_LINKER_OPTION_CREATE_LIBRARY = SPV_BIT(1),
```

Comment:
I’ll switch it to an opaque object, you are right.
If creating a library, I keep exported symbols, otherwise I remove them as we are producing an executable; I I have been mainly looking at it from an OpenCL’s viewpoint. I wanted to add more options, mainly the ones that can be passed to OpenCL in the linking step, like relaxing some conditions on floating point operations, but I haven’t looked into it yet.